From e482eb29038d3e79a5efca29bbb1beeccdbb7759 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Tue, 10 Mar 2020 15:27:36 +0700
Subject: [PATCH 1/2] arm64: defconfig: enable PREEMPT_RT config

This patch enables PREEMPT_RT_FULL config needed for RT kernel.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/configs/defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index d60c97d..2a67f4a 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -376,3 +376,6 @@ CONFIG_SATA_AHCI_PLATFORM=y
 CONFIG_INPUT_UINPUT=m
 CONFIG_INPUT_MOUSEDEV=m
 CONFIG_TMPFS_POSIX_ACL=y
+
+#Enable RT config
+CONFIG_PREEMPT_RT_FULL=y
-- 
2.7.4

